src/audio/google_audio_post_processing_mock.c

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2022 Google LLC.
+//
+// Author: Kehuang Li <kehuangli@google.com>
+
+#include "google_audio_post_processing.h"
+
+#include <stdint.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "sof/common.h"
+#include "sof/lib/alloc.h"
+#include "ipc/topology.h"
+
+#define GOOGLE_AUDIO_POST_PROCESSING_SAMPLE_RATE_HZ 48000
```

Comment:
This is second time define for the same, move to google_audio_post_processing.h?

Reply:
Actually, I did that on purpose. 48kHz is a general default, but it may not always be the case, so I'd prefer not to define it in the library API. Therefore, later if encountering special cases, e.g., 96kHz or 16kHz, we won't need to change the const in the API; instead, we can update the API to pass-in the sample rate and increase the API version number. WDYT?

Reply:
Yep that makes sense.